hadoop-ozone/dist/src/main/compose/ozonesecure-om-ha/test.sh

```diff
@@ -30,6 +30,8 @@ execute_robot_test scm kinit.robot
 
 execute_robot_test scm freon
 
+execute_robot_test scm basic/links.robot
```

Comment:
Out of curiosity: why adding this robot test to both

` hadoop-ozone/dist/src/main/compose/ozone-om-ha-s3/test.sh` and
`hadoop-ozone/dist/src/main/compose/ozonesecure-om-ha/test.sh`.

Will it just run the same test twice on each CI?

Reply:
Good question might be it is good to run only during security-enabled cluster.
But it has some tests which are bucket link features, so I thought it might be good to run in both secure and non-secure.

Reply:
I think it's useful to run the same test in different environments: secure and non-secure, non-HA and HA.  It increases coverage.  Also, non-secure versions provide some kind of baseline: if it works and secure one does not, we might know the problem is security-related.

If it takes too much time, we can tweak acceptance test splits (currently we have 3 concurrent runs in CI).

Reply:
Makes sense. Thanks for all your clarification!